Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -19,7 +43,21 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            var stop = 0;
+
+            if (array == null)
+            {
+                return;
+            }
+            
+            stop = array.Length / 2;
+            
```

Comment:
It would be nice if you could give a small explanation on algorithms. 

Reading algorithms takes time to understand and a comment could speed this up.

Reply:
oke, I will take care of it

Reply:
done